cli/command/trust/testdata/trust-revoke-prompt-termination.golden

```diff
@@ -0,0 +1,2 @@
+Please confirm you would like to delete all signature data for example/trust-demo? [y/N] 
+Aborting action.
```

Comment:
aah i guess this could be another confirmation prompt to "fix" by standardizing the output (aka not printing "Aborting action" and making it behave more like everything else). For another moment of course

Reply:
ah yeah true